src/task1/index.js

```diff
@@ -0,0 +1,5 @@
+const stringReverse = require('./stringReverse');
```

Comment:
Thanks for the note. I thought it is a good practice in js world to create index.js file for smth like package import.
I'll remove all index.js files
Thanks

Reply:
of course it is good practice to use index.js but for importing smth. In your case by importing (require('./stringReverse')) in index.js you just run this file, but do no import anything (at least because you do not export anything in stringReverse.js). If you want you may wrap code of ./stringReverse.js to fuction, export it. Then in index.js import it and run